docs/book/features/block_volume.md

```diff
@@ -375,3 +376,67 @@ Cluster
             VolumeAttributes:      type=vSphere CNS Block Volume
         Events:                <none>
     ```
+
+### Statically Provision a Block Volume in Tanzu Kubernetes Grid Cluster<a id="static_volume_provisioning_tkg"></a>
+
+You can statically create a block volume in a Tanzu Kubernetes Grid (TKG) Cluster using an unused PersistentVolumeClaim (PVC) from the Project Pacific cluster only if the latter satisfies the following conditions:
+
+- It is present in the same namespace as the TKG Cluster
+
+- It is associated with the same `StorageClass` as the default storage class in the TKG Cluster
+
+- Not attached to a Pod in the Project Pacific Cluster
```

Comment:
This doesnt exactly stop a user from statically provisioning a volume, its just that that volume can't be consumed by an application.

Reply:
Yes i agree but I assumed it will be good to mention here. Do you want me to explain that if the PVC is attached to a Pod on supervisor cluster, it cannot be consumed by an application in TKG cluster after provisioning? Or should I remove this completely?